pages/wizard/regions/task_step_bar.py

```diff
@@ -24,6 +24,22 @@
         'class': 'SetupType',
         'library': 'pages.wizard.rhev.setup_type',
     },
+    'Engine': {
+        'class': 'Engine',
+        'library': 'pages.wizard.rhev.engine',
+    },
+    'Hypervisor': {
+        'class': 'Hypervisor',
+        'library': 'pages.wizard.rhev.hypervisor',
+    },
+    'Configuration': {
```

Comment:
Initially, I had names like 'RHV*', so for 'Configuration' it would be 'RHVConfiguration', class would be the same and library 'rhv_configuration', if that helps.